source/renderer/app/stores/AssetsStore.js

```diff
@@ -56,21 +52,21 @@ export default class AssetsStore extends Store {
   } {
     return this.all.reduce((details, asset) => {
       const { policyId, assetName } = asset;
-      details[policyId + assetName] = asset;
+      details[`${policyId}${assetName}`] = asset;
       return details;
     }, {});
   }
 
-  getAssetDetails = (policyId: string, assetName: string): ?Asset =>
-    this.details[policyId + assetName];
+  getAssetDomain = (policyId: string, assetName: string): ?Asset =>
```

Comment:
Can you explain a bit what "domain" in `assetDomain` means? I see it in several parts in this PR but don't understand the meaning. Why not `getAsset` instead?

Reply:
I was trying to be more specific, as it was a vague `getDetails`, but it was indeed too much. ✔️